src/web/app/src/main/resources/geostore-spring-security.xml

```diff
@@ -169,6 +169,25 @@
 			</bean>
 		</constructor-arg>
 	</bean>
+
+
+	<!-- Uncomment the following to use Keycloak as a User and Group persistence layer.
+
+	<bean id="keycloakUserDAO" class="it.geosolutions.geostore.services.rest.security.keycloak.KeycloakUserDAO">
+		<constructor-arg ref="keycloakRESTClient"/>
+	</bean>
+	<alias name="keycloakUserDAO" alias="userDAO"/>
+
+	<bean id="keycloakUserGroupDAO" class="it.geosolutions.geostore.services.rest.security.keycloak.KeycloakUserGroupDAO">
+		<constructor-arg ref="keycloakRESTClient"/>
+	</bean>
+	<alias name="keycloakUserGroupDAO" alias="userGroupDAO"/>
+	
+	<alias name="externalSecurityDAO" alias="securityDAO"/>
+
+	-->
+
+
```

Comment:
replace with 
```suggestion
<!--  security itegration inclusions  -->
<import resource="classpath*:security-integration-${security.integration:default}.xml"/>
```

and add files for `secuirity-integration-default.xml` and `secuirity-integration-keycloak-direct.xml`